examples/example_mnist_grbm.py

```diff
@@ -1,76 +1,97 @@
 import os, sys, numpy, pandas, time
 
-#from paysage.backends import numba_engine as en
 from paysage import batch
 from paysage.models import hidden
 from paysage import fit
 from paysage import optimizers
-from paysage import backends as B
-
-import matplotlib.pyplot as plt
-import seaborn as sns
-
-def plot_image(image_vector, shape):
-    f, ax = plt.subplots(figsize=(4,4))
-    hm = sns.heatmap(numpy.reshape(image_vector, shape), ax=ax, cmap="gray_r", cbar=False)
-    hm.set(yticks=[])
-    hm.set(xticks=[])
-    plt.show(f)
-    plt.close(f)
+import plotting
```

Comment:
Added try/except clause on the relative import so that the script can be run from any directory:

~~~~
try:
  import plotting
except ImportError:
    from . import plotting
~~~~